sharding/node/node.go

```diff
@@ -103,7 +103,7 @@ func (n *shardingNode) Start() error {
 	// Starts every service attached to the sharding node.
 	for _, serviceFunc := range n.serviceFuncs {
 		// Initializes each service by passing in the node's cli context.
```

Comment:
// Initializes each service ~~by passing in the node's cli context~~ 
We are not passing in the cli context anymore